### ISSUE TITLE
Build fails on Node v6

### ISSUE THREAD

**Issue by stramel** [pikapkg/pack#22]
Got this error on Node v6

```
const unlink = util.promisify(_rimraf);
                    ^
TypeError: util.promisify is not a function
    at Object.<anonymous> (/home/travis/build/43081j/js-levenshtein/node_modules/@pika/pack/dist-node/index.js:1865:21)
    at Module._compile (module.js:577:32)
    at Object.Module._extensions..js (module.js:586:10)
    at Module.load (module.js:494:32)
    at tryModuleLoad (module.js:453:12)
    at Function.Module._load (module.js:445:3)
    at Module.require (module.js:504:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/travis/build/43081j/js-levenshtein/node_modules/@pika/pack/dist-node/index.bin.js:10:9)
    at Module._compile (module.js:577:32)
```

**Comment by stramel** [comment]
Not supported as stated in #14 

😅 	Forgot that we talked about that.

**Comment by FredKSchott** [comment]
np! If its alright with you, I'm going to keep this issue open to remind me (or anyone else who wants to tackle!) to add a "minNodeVersion" to `@pika/plugin-simple-bin` in the pipeline, now that the project has v8.x as a version support target.

**Comment by FredKSchott** [comment]
Fixed in d29095c
